tools/deploy_prow.sh

```diff
@@ -74,7 +74,7 @@ function launchMonitoring(){
 }
 
 function launchARC(){
-  kubectl apply -f https://github.com/actions/actions-runner-controller/releases/download/v0.27.1/actions-runner-controller.yaml --server-side
+  kubectl apply -f https://github.com/actions/actions-runner-controller/releases/download/v0.27.1/actions-runner-controller.yaml --server-side --force-conflicts
```

Comment:
Otherwise deploying this would fail with
```
Apply failed with 3 conflicts: conflicts with "kube-apiserver" using apiextensions.k8s.io/v1:
- .status.conditions
- .status.acceptedNames.kind
- .status.acceptedNames.plural
```